issues/usePatientInfo.ts

```diff
@@ -40,13 +72,38 @@ export default function usePatientInfo(
     );
 
   useEffect(() => {
-    userService.getUserInfo(userInfo.userId).then(setUserInfo);
+    userService.getUserInfo(userInfo.userId).then(() => {
+      if (isMounted()) {
+        setUserInfo;
+      }
+    });
+  }, [userId]); //should update userINfo State then other userEffects will render
 
+  useEffect(() => {
     logAccess('request-access', userInfo.userId);
     medicalRecordService
       .getMedicalRecord(userInfo.userId)
+<<<<<<< HEAD
```

Comment:
I don't know what's happened here, but for some reason the source code contains diff syntax.